PWGHF/D2H/Tasks/taskFlowCharmHadrons.cxx

```diff
@@ -375,17 +376,21 @@ struct HfTaskFlowCharmHadrons {
   void processDsMl(CollsWithQvecs::iterator const& collision,
                    CandDsDataWMl const& candidatesDs)
   {
-    runFlowAnalysis<DecayChannel::DsToKKPi, Partition<CandDsDataWMl>>(collision, selectedDsToKKPiWMl);
-    runFlowAnalysis<DecayChannel::DsToPiKK, Partition<CandDsDataWMl>>(collision, selectedDsToPiKKWMl);
+    auto candsDsToKKPiWMl = selectedDsToKKPiWMl->sliceByCached(aod::hf_cand::collisionId, collision.globalIndex(), cache);
+    auto candsDsToPiKKWMl = selectedDsToPiKKWMl->sliceByCached(aod::hf_cand::collisionId, collision.globalIndex(), cache);
+    runFlowAnalysis<DecayChannel::DsToKKPi, CandDsDataWMl>(collision, candsDsToKKPiWMl);
+    runFlowAnalysis<DecayChannel::DsToPiKK, CandDsDataWMl>(collision, candsDsToPiKKWMl);
```

Comment:
```suggestion
    runFlowAnalysis<DecayChannel::DsToKKPi>(collision, candsDsToKKPiWMl);
    runFlowAnalysis<DecayChannel::DsToPiKK>(collision, candsDsToPiKKWMl);
```
As I commented in https://github.com/AliceO2Group/O2Physics/pull/5089, the `T1` should be deduced from the `candidates` arguments. Providing it explicitly just opens doors for bugs.